fennel/test_lib/mock_client.py

```diff
@@ -426,6 +426,13 @@ def extract_features(
         for input_feature in input_feature_list:
             if isinstance(input_feature, Feature):
                 input_feature_names.append(input_feature.fqn_)
+            elif isinstance(input_feature, str):
+                if "." not in input_feature:
+                    raise Exception(
```

Comment:
Same here? Maybe have a function like `is_plausible_feature_fqn` and use that everywhere?